src/components/Workspace/SidePanel/TabContents.tsx

```diff
@@ -8,6 +13,47 @@ import { TabPanel } from './TabPanel'
  */
 
 export const getTabContents = (selectedIndex: number): JSX.Element[] => {
+  const apps: Record<string, CyApp> = useAppStore((state) => state.apps)
+
+  const getPanelComponents = () => {
+    const panels: any = []
+    Object.keys(apps).forEach((appId: string) => {
+      const app: CyApp = apps[appId]
+      const { components } = app
+      if (components === undefined) return
+
+      components.forEach((component: ComponentMetadata) => {
+        if (component.type === 'panel') {
```

Comment:
Minor improvement.  Maybe this could be a constant instead of a hard coded string?

Reply:
Thanks, I'll fix it.